src/components/skeletonView/index.tsx

```diff
@@ -342,7 +341,7 @@ class SkeletonView extends Component<InternalSkeletonViewProps, SkeletonState> {
     const {style, ...others} = this.props;
 
     return (
-      <View style={[styles.listItem, style]} {...this.getAccessibilityProps('Loading list item')} {...others}>
+      <View style={[styles.listItem, style]} {...this.listItemAccessibilityProps} {...others}>
```

Comment:
`useMemo` in `View` for _style will get recalculated every render because of new reference
`style={[styles.listItem, style]}`